### PR TITLE
✨ Rewrite of Amazon Music

### PR DESCRIPTION
**Pull request by StrikerFRFX**
So I've rewritten the Amazon Music Presence and it now shows what song is playing. 
The only thing I could not get working at all was the timestamps and recognizing the song was paused.
So I've removed the timestamps and the pausing logic because otherwise, it would constantly keep the time ticking even if they paused the song.

Screenshots:
2AWWfs66Gf
QqxVSYSF9r

### CODE REVIEW

**Comment by ririxidev** [comment]
resolve conflicts

**Review by Cairo2k18 — CHANGES_REQUESTED** [review]
Bump the presence version in the `metadata.json` file.

**Review by Bas950 — CHANGES_REQUESTED** [review]
Bump version

**Review by Bas950 — CHANGES_REQUESTED** [review]
Bump version

**Comment by StrikerFRFX** [comment]
> Bump version

It's bumped, from 1.2.6 to 1.2.7

**Comment by StrikerFRFX** [comment]
> Bump version

It's bumped, from 1.2.6 to 1.2.7

File: websites/A/Amazon Music/presence.ts [hunk 1]
```diff
@@ -6,51 +6,129 @@ const presence = new Presence({
...
+  timestamps;
+
+function getDuration(time1: string, time2: string) {
+  return (
+    parseInt(time1.split(":")[0]) * 60 +
+    parseInt(time1.split(":")[1]) +
+    parseInt(time2.split(":")[0]) * 60 +
+    parseInt(time2.split(":")[1])
+  );
 }
```
Comment (Bas950) [inline websites/A/Amazon Music/presence.ts:23]: image

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload new proof of it workin

**Comment by StrikerFRFX** [comment]
New Screenshots:
JKJKgOO6UD
0cokssFiqo
A9XnR5DwZ5

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**